### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by andreaslyn**
Attempt to fix the CI and close https://github.com/HoTT/HoTT/issues/1417

### CODE REVIEW

**Comment by mikeshulman** [comment]
Looks like the CI is happy!!

**Comment by Alizter** [comment]
Excellent!

File: .gitmodules [hunk 1]
```diff
@@ -7,5 +7,5 @@
 	url = https://github.com/JasonGross/coq-scripts.git
 [submodule "etc/coq-dpdgraph"]
 	path = etc/coq-dpdgraph
-	url = https://github.com/Karmaki/coq-dpdgraph.git
-	branch = coq-master
+	url = https://github.com/andreaslyn/coq-dpdgraph.git
```
Comment (spitters) [inline .gitmodules:10]: Very nice that the CI is working again.
Perhaps we want to push your improvements to dpdgraph to the main repo?
https://github.com/Karmaki/coq-dpdgraph

Comment (Alizter) [reply .gitmodules:10]: Turn's out I made very similar changes a while back, but they weren't included. The PR is still open: Karmaki/coq-dpdgraph#73. @ybertot is the maintainer I spoke to, but I can't remember why we didn't include it.

**Comment by ybertot** [comment]
The PR https://github.com/Karmaki/coq-dpdgraph/pull/73 has now been rebased and merged in coq-dpdgraph.  the coq-master branch of coq-dpdgraph is now compatible both with the master branch of coq and OCaml version 4.10.0

**Comment by ybertot** [comment]
Thanks for the reminder.

**Comment by spitters** [comment]
@Alizter or @andreaslyn shall we depend on the official version of dpdgraph again?

**Comment by Alizter** [comment]
@spitters Would be good to do, but we need to check it really works. @andreaslyn's pr is slightly different to mine.

**Pull request merged**